nntrainer/graph/network_graph.cpp

```diff
@@ -105,19 +105,18 @@ void NetworkGraph::countNonTrainableLayersAtBegin() {
 
 int NetworkGraph::realizeMultiInputType(
   const std::shared_ptr<LayerNode> &in_node) {
-  Layer &current = *in_node->getObject();
   int status = ML_ERROR_NONE;
   /**
    * Multi-input works with time distribution layer by itself
    *
    */
-  if (current.getNumInputs() == 1)
+  if (in_node->getNumInputs() <= 1)
```

Comment:
is this no error if it's empty? Wonder because we have __data__defined.

Reply:
yes, data is defined but it is not necessary for __data__ to be set till now.

Reply:
Okay